README.md

```diff
@@ -17,10 +17,10 @@ Run phpMyAdmin with Alpine, Apache and PHP FPM.
 [![mips64le build status badge](https://img.shields.io/jenkins/s/https/doi-janky.infosiftr.net/job/multiarch/job/mips64le/job/phpmyadmin.svg?label=mips64le)](https://doi-janky.infosiftr.net/job/multiarch/job/mips64le/job/phpmyadmin)
 [![ppc64le build status badge](https://img.shields.io/jenkins/s/https/doi-janky.infosiftr.net/job/multiarch/job/ppc64le/job/phpmyadmin.svg?label=ppc64le)](https://doi-janky.infosiftr.net/job/multiarch/job/ppc64le/job/phpmyadmin)
 [![s390x build status badge](https://img.shields.io/jenkins/s/https/doi-janky.infosiftr.net/job/multiarch/job/s390x/job/phpmyadmin.svg?label=s390x)](https://doi-janky.infosiftr.net/job/multiarch/job/s390x/job/phpmyadmin)
-[![Docker Pulls](https://img.shields.io/docker/pulls/_/phpmyadmin.svg)][hub]
-[![Docker Stars](https://img.shields.io/docker/stars/_/phpmyadmin.svg)][hub]
+[![Docker Pulls](https://img.shields.io/docker/pulls/_/phpmyadmin.svg)](https://hub.docker.com/_/phpmyadmin)
+[![Docker Stars](https://img.shields.io/docker/stars/_/phpmyadmin.svg)](https://hub.docker.com/_/phpmyadmin)
```

Comment:
This is one of those changes where it slightly adds complexity (due to needing to maintain the link in two places, the opposite of only doing things once), but overall improving the readability. I like this change.

Reply:
Yeah, I always found it fine to use URLs directly - they are meant to be stable identifiers.